src/protectionmarketlist.schema.json

```diff
@@ -0,0 +1,441 @@
+          "examples": ["Cozy USD Coin"]
+        },
+        "symbol": {
+          "type": "string",
+          "description": "The symbol for the protection market; must be alphanumeric",
+          "pattern": "^[a-zA-Z0-9+\\-%\\/\\$]+$",
+          "minLength": 1,
+          "maxLength": 100,
+          "examples": ["cozyUSDC"]
+        },
+        "logoURI": {
+          "type": "string",
+          "description": "A URI to the protection market's logo asset; if not set, interface will attempt to find a logo based on the token address; suggest SVG or PNG of size 64x64",
+          "format": "uri",
+          "examples": ["ipfs://QmXfzKRvjZz3u5JRgC4v5mGVbm9ahrUiB4DgzHBsnWbTMM"]
+        },
+        "investmentOpportunity": {
+          "$ref": "#/definitions/InvestmentOpportunity"
+        },
+        "isTriggered": {
+          "type": "boolean",
+          "description": "True if this market has been triggered, false if not triggered or not applicable",
+          "examples": [true, false]
+        },
+        "trigger": {
+          "type": "string",
+          "description": "The checksummed address of the market's trigger contract, or the zero address for markets without a trigger",
+          "pattern": "^0x[a-fA-F0-9]{40}$",
+          "examples": [
+            "0x02E57c5800a345Fe97d73fA8bB72B948E19aA499",
+            "0x0000000000000000000000000000000000000000"
+          ]
+        },
+        "underlying": {
+          "$ref": "#/definitions/TokenInfo"
+        },
+        "tags": {
+          "type": "array",
+          "description": "An array of tag identifiers associated with the market; tags are defined at the list level",
+          "items": {
+            "$ref": "#/definitions/TagIdentifier"
+          },
+          "maxItems": 10,
+          "examples": ["stablecoin", "compound"]
+        },
+        "extensions": {
+          "type": "object",
+          "description": "An object containing any arbitrary or vendor-specific market metadata",
+          "propertyNames": {
+            "$ref": "#/definitions/ExtensionIdentifier"
+          },
+          "additionalProperties": {
+            "$ref": "#/definitions/ExtensionValue"
+          },
+          "maxProperties": 10,
+          "examples": [
+            {
+              "color": "#000000",
+              "is_verified_by_me": true
+            }
+          ]
+        }
+      }
+    },
+    "StrategyLeg": {
+      "type": "object",
+      "description": "Describes one of the steps taken to enter an investment opportunity",
+      "properties": {
+        "action": {
+          "enum": ["Borrow", "Deposit to", "Stake on"],
+          "type": "string"
+        },
+        "destination": {
+          "type": "number",
+          "description": "Platform ID the strategy leg targets",
+          "minimum": 1
+        },
+        "logoURI": {
+          "type": "string",
+          "description": "A URI for the logo of the strategy leg; prefer SVG or PNG of size 256x256",
```

Comment:
May as well make the size of the logos match.

256x256 sound great... even better than 128x128 suggested above 😅

Reply:
https://github.com/Cozy-Finance/protection-market-lists/pull/1#discussion_r704692017